main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
     int toggleCenterTap();
-    // convert vibration to voltage
     double vibToVoltage(double vibrationPower = 0.0);
 };
-/*
-Thing 6) Neck
-5 properties:
-    1) Fretboard Material (std::string)
-    2) scale length in cm (double)
-    3) number of frets(int)
-    4) fret width in mm (float)
-    5) nut material (std::string)
-3 things it can do:
-    1) adjust truss rod
-    2) fret note
-    3) clean fretboard
- */
+
+void Pickup::adjustHeight(double heightAdjustment){ pickupDistance -= heightAdjustment;}
+
+int Pickup::toggleCenterTap()
+{
+    if (pickupType == "Humbucker") 
+    {
+        if (numCoils == 2) numCoils = 1;
+        else               numCoils = 2;
+    }
+
+    return numCoils;
+}
+
+double Pickup::vibToVoltage(double vibrationPower)
+{
+    return vibrationPower * pickupDistance;
+}
+
 struct Neck
 {
-    // Fretboard Material 
     std::string fretboardMaterial = "rosewood";
-    // scale length in cm 
     double scaleLength = 60.96;
-    // number of frets(int)
     int numFrets = 24;
-    // fret width in mm (float)
     float fretWidth = 0.94f;
-    // truss rod neck releief in mm
     float neckRelief = 0.25f;
-    //  fretboard soiled
     bool fretboardSoiled = false;
 
-    // adjust truss rod
     float adjustTrussRod(float numTurns = 0.0);
-    // fret note
     int fretNote(int stringNum = 6, int fretNum = 0);
-    // clean fretboard
     void cleanFretboard();
-
 };
-/*
-Thing 7)  Body
-5 properties:
-    1) body type(std::string)
-    2) color (std::string)
-    3) cutout radius in cm (float)
-    4) number of cutouts (int)
-    5) thickness in cm (float)
-3 things it can do:
-    1) select pickup output
-    2) adjust pickup volume
-    3) adjust pickup tone
- */
+
+float Neck::adjustTrussRod(float numTurns)
+{
+    neckRelief *= numTurns * 0.1f;
+    return neckRelief;
+}
+
+int Neck::fretNote(int stringNum, int fretNum)
+{
+    return stringNum + fretNum;
+}
+
+void Neck::cleanFretboard() { fretboardSoiled = false;}
+
 struct Body
 {
-    // body type
     std::string bodyType = "solid";
-    // color 
     std::string color = "sunburst";
-    // cutout radius in cm 
     float cutoutRadius = 2.5f;
-    // number of cutouts 
     int numCutouts = 2;
-    // thickness in cm 
     float thickness = 4.76f;
 
-    // select pickup output
     int selectPickup(int pickup);
-    // adjust pickup volume
     float changeVolume(float newVolume);
-    // adjust pickup tone
     float changeTone(float newTone);
 };
-/*
-Thing 8) Tuners
-5 properties:
-    1) tuner metal (std::string)
-    2) string tension in pounds (double)
-    3) number of tuners(int)
-    4) tuner angle in degrees (double)
-    5) tuner gear ratio (float)
-3 things it can do:
-    1) rotate tuner key
-    2) adjust string tension
-    3) tighten set nut
- */ 
+
+int Body::selectPickup(int pickup) { return pickup;}
+
+float Body::changeVolume(float newVolume) { return newVolume;}
+
+float Body::changeTone(float newTone){ return newTone;}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise